<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCourseWidget</class>
 <widget class="QWidget" name="AddCourseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>增加课程</string>
  </property>
  <widget class="QLineEdit" name="lineName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTeacher">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbCap">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbType">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>专业课</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>非专业课</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>课程名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTeacher">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>授课教师</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCap">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>上限人数</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelType">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>课程类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnConfirm</sender>
   <signal>clicked()</signal>
   <receiver>AddCourseWidget</receiver>
   <slot>confirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>confirm()</slot>
 </slots>
</ui>
